### ISSUE TITLE
Why is participation ratio rounded to an integer?

### ISSUE THREAD

**Issue by RylanSchaeffer** [scikit-learn-contrib/scikit-dimension#12]
I just noticed that participation ratio is always an integer. Why is that?

https://github.com/scikit-learn-contrib/scikit-dimension/blob/ea1a4954fdb6ce6e8b36bd9597044cd4040149ab/skdim/id/_PCA.py#L189

**Comment by j-bac** [comment]
Yes it is rounded. If you think it is more useful or standard not to round I can make the change

**Comment by RylanSchaeffer** [comment]
I've never heard of it being rounded, nor had two of my labmates. We also tried to find whether anyone recommends rounding it in a paper, but didn't.

Overall, it's not a big deal. We were just perplexed why we were getting integer values. Maybe change it?

**Comment by j-bac** [comment]
Just updated, thanks for the suggestion. Will release on pypi after some other updates

**Comment by RylanSchaeffer** [comment]
@j-bac any updates on the new release?

**Comment by j-bac** [comment]
my bad, I did make a new release but seems it didn't include the change. Just added v0.3.1 on pypi

**Comment by RylanSchaeffer** [comment]
I think that change introduced an error. Because `de` is now now longer an integer, this indexing breaks:

https://github.com/scikit-learn-contrib/scikit-dimension/blob/ea1a4954fdb6ce6e8b36bd9597044cd4040149ab/skdim/id/_PCA.py#L192

**Comment by RylanSchaeffer** [comment]
The error:

```
python3.7/site-packages/skdim/id/_PCA.py", line 192, in _participation_ratio
    return de, gaps[de - 1]
IndexError: only integers, slices (`:`), ellipsis (`...`), numpy.newaxis (`None`) and integer or boolean arrays are valid indices

```

**Comment by j-bac** [comment]
Thanks, just fixed this (0.3.2). I'm returning all gaps now under self.gap_
